### ISSUE TITLE
What's purpose of it?

### ISSUE THREAD

**Issue by dimka11** [openai/DALL-E#50]
If it doesn't include an image generator. What are the tasks where I can use this project?

**Comment by ccpaco** [comment]
Hi Dimka, I am possibly wrong but I think DALL-E does not have a public API and so it can't be used for image generation like it is used here [https://openai.com/blog/dall-e/](url)

The only public API from OpenAI is GPT-3, which mostly for text based programming. 

[https://openai.com/api/](url)

**Comment by jpryne** [comment]
I can understand wanting to keep this private, but then it's not really open now is it?

**Comment by wellsilver** [comment]
Yes, we would like to try dall e IDC if I have to use my own GPU
